pvlib/pvsystem.py

```diff
@@ -1659,12 +1665,19 @@ def sapm(effective_irradiance, temp_cell, module):
     temperature.sapm_module
     '''
 
-    T0 = 25
+    # TODO: remove this check after deprecation period for change in effective
+    # irradiance units, made in v0.7
```

Comment:
we'll need to make a follow up issue for this and label it with the appropriate milestone